config.go

```diff
@@ -35,6 +35,18 @@ type ServerConfig struct {
 	CleanDiskRunPeriod   Duration `toml:"clean_disk_run_period"`
 	CleanDiskFileTTL     Duration `toml:"clean_disk_file_ttl"`
 	CleanDeviceRunPeriod Duration `toml:"clean_device_run_period"`
+	// Auto-drain period is a window that only certain number of servers are allowed to run auto-drain operations.
+	// When the period advances to the next one, servers currently running auto-drain will stop and some other
+	// servers are going to run auto-drain.
+	AutoDrainRunPeriod Duration `toml:"auto_drain_run_period"`
```

Comment:
@cenkalti how to find appropriate value for this config?

Let's say;

- We have 100 servers
- 50 of them exceeds `AutoDrainThreshold`
- `AutoDrainDeviceRatio` is set to `10`

In this case, if we set `AutoDrainRunPeriod` as `1 hour`, `5` servers will be drained every 1 hour. So, auto-drain will be finished in 10 hours for  `50` servers. Is the calculation correct?

If there is small difference for the servers those exceeds `AutoDrainThreshold` and those don't, 1 hour might be a high value. Maybe we should use less than that? Or should we find appropriate `AutoDrainRunPeriod` config with run & monitor method?